modules/library/referencing/src/main/java/org/geotools/referencing/datum/DefaultEllipsoid.java

```diff
@@ -496,4 +499,22 @@ protected String formatWKT(final Formatter formatter) {
         formatter.append(Double.isInfinite(ivf) ? 0 : ivf);
         return "SPHEROID";
     }
+
+    @Override
+    public String formatPROJ(final PROJFormatter formatter) {
+        final double ivf = getInverseFlattening();
+        if (!formatter.isDatumProvided()) {
+            if (formatter.isEllipsoidProvided()) {
+                return "+ellps=";
```

Comment:
Makes me wonder if it would not be better to have something like:

```
formatter.append("ellps", <name of ellipsoid here>);
```

Reply:
the formatting may involve nested objects which may add additional string definition.
At that phase you may not know yet the name of the ellipsoid (or if an alias is actually needed)